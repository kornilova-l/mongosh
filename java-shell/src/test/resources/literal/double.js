// command
0.1