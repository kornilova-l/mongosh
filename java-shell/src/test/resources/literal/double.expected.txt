DoubleResult: 0.1